Configs/.config/waybar/config.jsonc

```diff
@@ -203,9 +224,9 @@
         "rotate": 0,
         "format-muted": "婢",
         "on-click": "pavucontrol -t 3",
-        "on-click-middle": "~/.config/hyprdots/scripts/volumecontrol.sh -o m",
-        "on-scroll-up": "~/.config/hyprdots/scripts/volumecontrol.sh -o i",
-        "on-scroll-down": "~/.config/hyprdots/scripts/volumecontrol.sh -o d",
+        "on-click-middle": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o m",
+        "on-scroll-up": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o i",
+        "on-scroll-down": "/home/alex/.config/hyprdots/scripts/volumecontrol.sh -o d",
```

Comment:
You did a oops here, this was better before, idk everyone here is alex :D

Reply:
ok i will change it i forgot to see there